### PR TITLE
Tweaking READMEs

### PR DESCRIPTION
**Pull request by ACStone-MTS**
Moving the READMEs around and fixing some small errors found during Zoom reviews.

### CODE REVIEW

**Review by darthsiroftardis — APPROVED** [review]

**Review by ipopescu — COMMENTED** [review]
@ACStoneCL @darthsiroftardis 
Can we change the Ownership table to say this, and thus be consistent with the rest of the tables and modalities?

| Ownership         | u8  |
|--------------|-----|
| Minter       | 0   |
| Assigned     | 1   |
| Transferable | 2   |

**Comment by ACStone-MTS** [comment]
> @ACStoneCL @darthsiroftardis Can we change the Ownership table to say this, and thus be consistent with the rest of the tables and modalities?
> 
> Ownership	u8
> Minter	0
> Assigned	1
> Transferable	2

I don't see any reason why not, I've made the change.

**Comment by ipopescu** [comment]
Thanks @ACStoneCL!

**Comment by darthsiroftardis** [comment]
> @ACStoneCL @darthsiroftardis Can we change the Ownership table to say this, and thus be consistent with the rest of the tables and modalities?
> Ownership 	u8
> Minter 	0
> Assigned 	1
> Transferable 	2

Thanks for the catch

File: README.md [hunk 1]
```diff
@@ -293,7 +301,182 @@ folder within the project folder.
...
+```bash
+
+casper-client --put-deploy -n http://localhost:11101/rpc --chain-name "casper-net-1" --payment-amount 500000000000 -k ~/casper/casper-node/utils/nctl/assets/net-1/nodes/node-1/keys/secret_key.pem --session-path ~/casper/enhanced-nft/contract/target/wasm32-unknown-unknown/release/contract.wasm 
+--session-arg "collection_name:string='CEP-78-collection'" 
+--session-arg "collection_symbol:string='CEP78'" 
+--session-arg "total_token_supply:u64='100'" 
+--session-arg "ownership_mode:u8='2'" 
+--session-arg "nft_kind:u8='1'" 
+--session-arg "nft_metadata_kind:u8='0'" 
+--session-args-json '[{"name":"token_meta_data","type":"String","value":"{\"name\": \"John Doe\",\"token_uri\": \"https:\/\/www.barfoo.com\",\"checksum\": \"940bffb3f2bba35f84313aa26da09ece3ad47045c6a1292c2bbd2df4ab1a55fb\"}"},{"name":"token_owner", "type":"Key","value":"account-hash-e9ff87766a1d2bab2565bfd5799054946200b51b20c3ca7e54a9269e00fe7cfb"},{"name":"nft_contract_hash","type":"Key","value":"hash-206339c3deb8e6146974125bb271eb510795be6f250c21b1bd4b698956669f95"}]'
```
Comment (darthsiroftardis) [inline README.md]: Can we change this here as well please

File: README.md [hunk 2]
```diff
@@ -247,6 +255,9 @@ The `MetadataMutability` option of `Mutable` cannot be used in conjunction with
 #### Installing the contract.
 
 The `main.rs` file within the contract provides the installer for the NFT contract. Users can compile the contract to Wasm using the `make build-contract` with the provided Makefile.
+
```
Comment (darthsiroftardis) [inline README.md:258]:
```suggestion
The pre-built Wasm for the contract and all other utility session code can be found as part of the most current release. User's wishing to build the Wasm themselves can pull the code and the `make build-contract` provided in the included Makefile. Please not, however, as part of building the contract, you will need to install `wasm-strip`.
```

File: README.md [hunk 3]
```diff
@@ -247,6 +255,9 @@ The `MetadataMutability` option of `Mutable` cannot be used in conjunction with
...
+If you wish to build the Wasm yourself, you must first install `wasm-strip`.
```
Comment (darthsiroftardis) [inline README.md]: Can we remove this in favor of the suggestion I've provided

**Review by darthsiroftardis — APPROVED** [review]
Optimistic approval pending nits

File: README.md [hunk 4]
```diff
@@ -293,7 +304,182 @@ folder within the project folder.
...
+
+
+The session arguments match the available modalities as listed in the main [README](https://github.com/casper-ecosystem/cep-78-enhanced-nft).
+
+<details>
+<summary><b>Casper client command without comments</b></summary>
+
+```bash
+
+casper-client --put-deploy -n http://localhost:11101/rpc --chain-name "casper-net-1" --payment-amount 500000000000 -k ~/casper/casper-node/utils/nctl/assets/net-1/nodes/node-1/keys/secret_key.pem --session-path ~/casper/enhanced-nft/contract/target/wasm32-unknown-unknown/release/contract.wasm
```
Comment (darthsiroftardis) [inline README.md]:
```suggestion
casper-client put-deploy -n http://localhost:11101/rpc --chain-name "casper-net-1" --payment-amount 500000000000 -k ~/casper/casper-node/utils/nctl/assets/net-1/nodes/node-1/keys/secret_key.pem --session-path ~/casper/enhanced-nft/contract/target/wasm32-unknown-unknown/release/contract.wasm 
```

File: README.md [hunk 5]
```diff
@@ -293,7 +304,182 @@ folder within the project folder.
...
+
+```
+
+</details>
+
+#### Minting an NFT
+
+Below is an example of a `casper-client` command that uses the `mint` function of hte contract to mint an NFT for the user associated with `node-1` in an [NCTL environment](https://docs.casperlabs.io/dapp-dev-guide/building-dapps/nctl-test/).
+
+* `casper-client --put-deploy -n http://localhost:11101/rpc --chain-name "casper-net-1" --payment-amount 500000000000 -k ~/casper/casper-node/utils/nctl/assets/net-1/nodes/node-1/keys/secret_key.pem --session-path ~/casper/enhanced-nft/client/mint_session/target/wasm32-unknown-unknown/release/mint_call.wasm`
```
Comment (darthsiroftardis) [inline README.md]:
```suggestion
* `casper-client put-deploy -n http://localhost:11101/rpc --chain-name "casper-net-1" --payment-amount 500000000000 -k ~/casper/casper-node/utils/nctl/assets/net-1/nodes/node-1/keys/secret_key.pem --session-path ~/casper/enhanced-nft/client/mint_session/target/wasm32-unknown-unknown/release/mint_call.wasm`
```

File: README.md [hunk 6]
```diff
@@ -293,7 +304,182 @@ folder within the project folder.
...
+    Metadata describing the NFT to be minted, passed in as a `string`.
+
+
+
+<details>
+<summary><b>Casper client command without comments</b></summary>
+
+```bash
+
+casper-client --put-deploy -n http://localhost:11101/rpc --chain-name "casper-net-1" --payment-amount 500000000000 -k ~/casper/casper-node/utils/nctl/assets/net-1/nodes/node-1/keys/secret_key.pem --session-path ~/casper/enhanced-nft/client/mint_session/target/wasm32-unknown-unknown/release/mint_call.wasm
```
Comment (darthsiroftardis) [inline README.md]:
```suggestion
casper-client put-deploy -n http://localhost:11101/rpc --chain-name "casper-net-1" --payment-amount 500000000000 -k ~/casper/casper-node/utils/nctl/assets/net-1/nodes/node-1/keys/secret_key.pem --session-path ~/casper/enhanced-nft/client/mint_session/target/wasm32-unknown-unknown/release/mint_call.wasm
```

File: README.md [hunk 7]
```diff
@@ -293,7 +304,182 @@ folder within the project folder.
...
+
+```
+
+</details>
+
+#### Transferring NFTs Between Users
+
+Below is an example of a `casper-client` command that uses the `transfer` function to transfer ownership of an NFT from one user to another. In this case, we are transferring the previously minted NFT from the user associated with `node-2` to the user associated with `node-3`.
+
+* `casper-client --put-deploy -n http://localhost:11101/rpc --chain-name "casper-net-1" --payment-amount 500000000000 -k ~/casper/casper-node/utils/nctl/assets/net-1/nodes/node-2/keys/secret_key.pem --session-path ~/casper/enhanced-nft/client/transfer_session/target/wasm32-unknown-unknown/release/transfer_call.wasm`
```
Comment (darthsiroftardis) [inline README.md]:
```suggestion
* `casper-client put-deploy -n http://localhost:11101/rpc --chain-name "casper-net-1" --payment-amount 500000000000 -k ~/casper/casper-node/utils/nctl/assets/net-1/nodes/node-2/keys/secret_key.pem --session-path ~/casper/enhanced-nft/client/transfer_session/target/wasm32-unknown-unknown/release/transfer_call.wasm`
```

File: README.md [hunk 8]
```diff
@@ -293,7 +304,182 @@ folder within the project folder.
...
+5) `--session-arg "token_id:u64='0'"` 
+
+    The `token_id` of the NFT to be transferred.
+
+<details>
+<summary><b>Casper client command without comments</b></summary>
+
+```bash
+
+casper-client --put-deploy -n http://localhost:11101/rpc --chain-name "casper-net-1" --payment-amount 500000000000 -k ~/casper/casper-node/utils/nctl/assets/net-1/nodes/node-2/keys/secret_key.pem --session-path ~/casper/enhanced-nft/client/transfer_session/target/wasm32-unknown-unknown/release/transfer_call.wasm
```
Comment (darthsiroftardis) [inline README.md]:
```suggestion
casper-client put-deploy -n http://localhost:11101/rpc --chain-name "casper-net-1" --payment-amount 500000000000 -k ~/casper/casper-node/utils/nctl/assets/net-1/nodes/node-2/keys/secret_key.pem --session-path ~/casper/enhanced-nft/client/transfer_session/target/wasm32-unknown-unknown/release/transfer_call.wasm 
```

File: README.md [hunk 9]
```diff
@@ -293,7 +304,182 @@ folder within the project folder.
...
+    The entry point corresponding to the `burn` function.
+
+3) `--session-arg "token_id:u64='1'"`
+
+    The token ID for the NFT to be burned. If the `identifier_mode` is not set to `Ordinal`, you must provide the `token_hash` instead.
+
+<details>
+<summary><b>Casper client command without comments</b></summary>
+
+casper-client --put-deploy -n http://localhost:11101/rpc --chain-name "casper-net-1" --payment-amount 500000000000 -k ~/casper/casper-node/utils/nctl/assets/net-1/nodes/node-1/keys/secret_key.pem
```
Comment (darthsiroftardis) [inline README.md]:
```suggestion
casper-client put-deploy -n http://localhost:11101/rpc --chain-name "casper-net-1" --payment-amount 500000000000 -k ~/casper/casper-node/utils/nctl/assets/net-1/nodes/node-1/keys/secret_key.pem
```

**Review by ipopescu — COMMENTED** [review]
~~I see NFTMetadataKind can be NFT721. I cannot find info on NFT721, do you mean ERC721?~~ I'll resolve this internally.

**Pull request merged**